build/src/src/dyndnsClient/getKeys.js

```diff
@@ -0,0 +1,68 @@
+const EthCrypto = require('eth-crypto');
+const fs = require('file-system');
+const util = require('util');
+const logs = require('../logs.js')(module);
+
+/**
+ * EthCrypto reference
+ *
+ * - Create private key
+ * const identity = EthCrypto.createIdentity();
+ * {
+      address: '0x3f243FdacE01Cfd9719f7359c94BA11361f32471',
+      privateKey: '0x107be946709e41b7895eea9f2dacf998a0a9124acbb786f0fd1a826101581a07',
```

Comment:
Avoid writing complete private public keys even in the comments

Reply:
Using '0x107be9...'